main/main.go

```diff
@@ -409,6 +411,13 @@ func replicateAction(c *cli.Context) error {
 	batcherConfig[config.VAR_NAME_BATCHER_MEMORY_SOFT_LIMIT] = batcherMemorySoftLimit
 	batcherConfig[config.VAR_NAME_BATCHER_ROUTING_METHOD] = batcherRoutingMethod
 
+	if c.GlobalInt(config.VAR_NAME_START_SLEEP) > 0 {
+		log.Info(
+			fmt.Sprintf(
+				"Sleeping for %d seconds before starting replicate...", c.GlobalInt(config.VAR_NAME_START_SLEEP)))
```

Comment:
I understand why you did this but let's discuss if we can do this without a sleep since it just avoids the race condition and doesn't fix it.

Reply:
This isn't used anymore. If it's logic we never intend to ever use, I can rip it out. I can see how if we need to use it then we're not doing something right. Thoughts?

Reply:
Yeah if it's not used then remove.